specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/DatabaseAdvancedThreatProtection.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
Can you add the rest of the resource types to this API version?

Reply:
As I understand it, they want to push this API version already. I will not make it on time to add the other resource types... I consulted with PMs and leads, and it is OK by them to have Server and Database in this API version and Managed Instance in the next one.

Reply:
Clarified that the rest of the APIs from the previous version will be ported by someone else on the SQL team before raising a PR to main